gosec/gosec_annotations.js

```diff
@@ -15,7 +16,8 @@ function getAnnotation (issue, directory) {
   const endLine = startLine
   const annotationLevel = getAnnotationLevel(issue.severity, issue.confidence)
   const title = `${issue.rule_id}: ${issue.details}`
-  const message = `The issue is in the code: ${issue.code}`
+  const message = title === config.syntaxErrorTitle ? `${issue.code}`
+    : `The issue is in the code: ${issue.code}`
```

Comment:
`The issue is in the code` sounds awkward and obvious.  In your second example, what is the issue?

Reply:
I used the prefix: `The issue is in the code` to annotate where exactly in the line is the problem. Bandit gives you information about the exact code causing the problem so do Gosec and I wanted to use that.

Reply:
I say that because we already know it's an issue and we already know its source code.  So sounds awkward in that way.

Reply:
Okay, then I can remove that.